backend/app/usecase/repository/shortlink_fake.go

```diff
@@ -105,6 +105,22 @@ func (s ShortLinkFake) UpdateShortLink(oldAlias string, shortLinkInput entity.Sh
 	}, nil
 }
 
+// DeleteShortLink deletes an existing user short link.
+func (s ShortLinkFake) DeleteShortLink(input entity.ShortLinkInput) error {
+	alias := input.GetCustomAlias("")
+
+	if alias == "" {
+		return errors.New("empty alias")
+	}
```

Comment:
```suggestion
        if shortLinkInput.CustomAlias == nil {
		return entity.ShortLink{}, errors.New("alias empty")
	}
	
	alias := shortLinkInput.GetCustomAlias("")
```
I agree with @arberiii. Let the use case decide whether it considers empty string as an empty alias. But I think `null` alias should be considered as empty alias at the repo level, and the `UpdateShortLink` method has a check like this already.

Reply:
This is not needed after we change the parameter from `ShortLinkInput` to `alias string`